tests/functional/adapter/materialized_view_tests/test_materialized_views.py

```diff
@@ -70,15 +70,9 @@ def test_materialized_view_create_idempotent(self, project, my_materialized_view
         )
         assert self.query_relation_type(project, my_materialized_view) == "materialized_view"
 
-    @pytest.mark.skip(
-        "The current implementation does not support overwriting materialized views with tables."
-    )
     def test_table_replaces_materialized_view(self, project, my_materialized_view):
```

Comment:
These tests can be removed entirely since they're inheriting from `dbt-core`. Ultimately if everything is implemented, then these test classes should just have configuration and no test cases.